docs/kratos/emails-sms/10_sending-sms.mdx

```diff
@@ -0,0 +1,92 @@
+---
+id: sms-messages
+title: Configure SMS message courier
+sidebar_label: Sending SMS messages
+---
```

Comment:
Author's comment - I haven't really done anything to this doc aside from giving it a new ID and title. We can work on it in the near future.